docs/models/overrides.md

```diff
@@ -37,15 +41,18 @@ The format of an override can be seen on the [wiki][format], and a good example
 }
 ```
 
-And here's a hypothetical snippet from the supporting code. (This does not have to be client-only; it will work on a server too. In vanilla, properties are registered in the item's constructor.)
+And here's a hypothetical snippet from the supporting code. Unlike the older versions (lower than 1.16.x), this needs to be done on client side only because ItemModelsProperties does not exist on server.
```

Comment:
Another minor formatting adjustment:
```suggestion
And here's a hypothetical snippet from the supporting code. Unlike the older versions (lower than 1.16.x), this needs to be done on client side only because `ItemModelsProperties` does not exist on server.
```